app/assets/javascripts/discourse/app/components/sidebar/api-sections.js

```diff
@@ -0,0 +1,23 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import { getOwner, setOwner } from "@ember/application";
+
+export default class SidebarApiSections extends Component {
+  @service sidebarState;
+
+  get sections() {
+    if (this.sidebarState.combinedMode) {
+      this.allSections = this.sidebarState.panels
+        .map((panel) => panel.sections)
+        .flat();
+    } else {
+      this.allSections = this.sidebarState.currentPanel.sections;
+    }
+
+    return this.allSections.map((customSection) => {
+      const section = new customSection();
```

Comment:
This is a regression from the previous implementation because we're initialising an object in a getter again.